senza/manaus/cloudformation.py

```diff
@@ -0,0 +1,154 @@
+from typing import Optional, List, Dict, Iterator
+from collections import OrderedDict
+from datetime import datetime
+from enum import Enum
+import boto3
+
+from .route53 import Route53
+
+
+class ResourceType(str, Enum):
+    route53_record_set = 'AWS::Route53::RecordSet'
+
+
+class CloudFormationStack:
+
+    """
+    A single AWS Cloudformation Stack.
+
+    See:
+    http://boto3.readthedocs.io/en/latest/reference/services/cloudformation.html
+    """
+
+    def __init__(self,
+                 stack_id: str,
+                 name: str,
+                 description: Optional[str],
+                 parameters: Dict[str, str],
+                 creation_time: datetime,
+                 last_updated_time: Optional[datetime],
+                 status: str,
+                 stack_status_reason: Optional[str],
+                 disable_rollback: bool,
+                 notification_arns: List[str],
+                 timeout_in_minutes: Optional[int],
+                 capabilities: Optional[List[str]],
+                 outputs: Optional[List[Dict]],
+                 tags: Dict[str, str],
+                 *,
+                 region: Optional[str]):
```

Comment:
I don't want region set by accident, since it's not returned by AWS, it's useful information but comes from the HostedZone when searching for domains inside it.